### DISCUSSION TITLE
再求（私密马赛各位）

### DISCUSSION THREAD

**Discussion by srtyhvff** [share121/inter-knot#4836] [委托]
绳网app上能用米社表情吗，脚本也下了，但就是用不了

**Comment by March-777th** [comment]
![Screenshot_2024-09-13-18-46-36-32_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/cae20863-a30b-486b-8088-e9ab05528811)
首先进入via设置界面启用脚本
然后点击这个[安装/更新脚本](https://dave-12138.cn/static/Tampermonkey/miyoushe-emot-to-github.user.js)
![Screenshot_2024-09-13-18-48-36-38_a252b927494330cdc2c8ba3b3f952e5e](https://github.com/user-attachments/assets/13132d65-7609-4bc7-9e18-f072d5a82bfe)
等待自动解析弹窗
之后进入github界面如果还没有的话就清除缓存，再次登陆刷新网页

**Reply by Robin-txdy** [reply]
记住一定要在via里面进入绳网，不然没用（不要问我为什么，说多了都是泪）<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >

**Comment by Dave-12138** [comment]
APP？APP不能——
绳网APP没有发帖功能，你用的是github的文本编辑器<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >
如果是githubAPP当然也不能用脚本，毕竟脚本是给浏览器用的

**Comment by fzdqz** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/09/04/a9fbeee1e21cf1abcf7bf2e610ee91f5_8437422338490524317.png" alt="玛拉妮-气氛组" width="75" >用via浏览器下载脚本就好了
